### DISCUSSION TITLE
Export some test utils

### DISCUSSION THREAD

**Discussion by Moumouls** [MasterKale/SimpleWebAuthn#82] [Ideas]
For an easier server integration testing using this lib i think it could be helpful to export some assertion/attestation examples (output of startAssertion/startAttestation).
What do you think about that ?

**Comment by MasterKale** [comment]
That's an interesting proposal. This could involve moving the various sample attestations and assertions used in the various .test.ts files into a separate testing-oriented package 🤔

**Comment by Moumouls** [comment]
Yes testing package will be the right solution !
Here i only first think about the `server` integration testing part

But we can imagine later a plug and play testing util function for E2E front testing (ex: for cypress)

**Reply by Moumouls** [reply]
@MasterKale if you can init the `@simplewebauthn/testing` package i can work on it tomorrow ;) a `testing-package` branch could be a good start. i think i will wait the testing package before working on the parse implementation

**Comment by Moumouls** [comment]
@MasterKale do you have an idea when you could have some time to initialize the testing package folder, since you well know which architecture you want for the mono repo :)

**Comment by MasterKale** [comment]
@Moumouls Check out #84, I'll merge it in later today :v:

**Comment by Moumouls** [comment]
So i'm back, parse server is now ready to receive the WebAuthn Auth Adapter via: https://github.com/parse-community/parse-server/pull/7052

i will push a draft PR at the end of the day (french hour), since you are in CA you will probably be able to take a look at it in the afternoon. 👍
